src/types/commands.ts

```diff
@@ -537,6 +537,14 @@ export interface ActivatePaintFormatCommand {
   type: "ACTIVATE_PAINT_FORMAT";
 }
 
+export interface CancelPaintFormatCommand {
+  type: "CANCEL_PAINT_FORMAT";
+}
+
+export interface KeepPaintFormatCommand {
+  type: "KEEP_PAINT_FORMAT";
+}
```

Comment:
Just add a boolean `persistent: true` in `ACTIVATE_PAINT_FORMAT`, dispatching 2 commands isn't needed and probably a bad idea